recipes/theano-pymc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "1.0.5" %}
+
+package:
+  name: theano-pymc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/T/Theano-PyMC/Theano-{{ version }}.tar.gz
+  sha256: bb566fa62f7923351299dfcb8947e42523057df65e81be444eb8ef0c6e602bf4
+  patches:
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - setuptools
+    - six >=1.9.0
+    - numpy >=1.9.1
+    - scipy >=0.14
+    - pygpu >=0.7.0,<0.8
+
+test:
+  imports:
+    - theano
+    - theano.compile
+    - theano.compile.sandbox
+    - theano.compile.tests
+    - theano.gof
+    - theano.gof.tests
+    - theano.gpuarray
+    - theano.gpuarray.dnn
+    - theano.gpuarray.linalg
+    - theano.misc
+    - theano.sandbox
+    - theano.scalar
+    - theano.scalar.tests
+    - theano.sparse
+    - theano.sparse.tests
+    - theano.tensor
+    - theano.tensor.nnet
+    - theano.tensor.nnet.tests
+    - theano.tensor.signal
+    - theano.tensor.signal.tests
+    - theano.tensor.tests
+    - theano.tests
```

Comment:
Add `commands` test for entry point:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Sorry, not quite sure what I need to do here. Could you provide a bit more info?

Reply:
And then if `theano-cache` supports a "help" mode:

```
test:
   import:
      - theano
      ...
   commands:
    - theano-cache --help
```

Reply:
Done.